shared/src/sys/out.rs

```diff
@@ -57,30 +57,37 @@ pub mod crypto {
 }
 
 pub mod vm {
-    use crate::clock::ChainEpoch;
     use crate::sys::TokenAmount;
     use crate::{ActorID, MethodNum};
 
     #[derive(Debug, Copy, Clone)]
     #[repr(packed, C)]
-    pub struct InvocationContext {
-        /// The value that was received.
-        pub value_received: TokenAmount,
+    pub struct SyscallMessageContext {
```

Comment:
```suggestion
    pub struct MessageContext {
```

?

Reply:
I would like to keep it consistent with `SyscallNetworkContext`

Reply:
I mean, rename both.